tests/test_utils.py

```diff
@@ -4,11 +4,13 @@
 
 import pytest
 from botocore.stub import ANY, Stubber
```

Comment:
Not sure the best place for this TODO, but I just want to leave it somewhere to make it clear to our future selves why there's a mix of `botocore.stub` and `moto` in this test file.
```suggestion
from botocore.stub import ANY, Stubber  # TODO: fully transition to moto for S3 mocking
```